Common/GeomPorting.cs

```diff
@@ -54,7 +60,7 @@ public static Vector3 CreateTranslationTarget(Vector3 to, Vector3 referencePoint
     /// </summary>
     /// <param name="v">Vector to be reflected</param>
```

Comment:
Add . at the end. Its missing in multiple places. However it's not a big deal so you can also skip this comment